brave/browser-laptop
# small ledger fixes

Auditor: @bbondy 
Test Plan:
1. Close the browser
2. Disable your Internet connection
3. Open the browser
4. Go to about:preferences#payments and see this:
![1](https://cloud.githubusercontent.com/assets/400552/20326121/46cb064a-ab3c-11e6-9cb8-f961273edef5.png)

Enable your Internet connection and a few seconds later see this:
![2](https://cloud.githubusercontent.com/assets/400552/20326130/56248c4c-ab3c-11e6-86ba-80f8a2df2c66.png)

Fixes https://github.com/brave/browser-laptop/issues/5606

Fixes https://github.com/brave/browser-laptop/issues/5503
